plugins/package-managers/bazel/src/main/kotlin/CompositeBazelModuleRegistryService.kt

```diff
@@ -47,7 +47,9 @@ internal class CompositeBazelModuleRegistryService(
          * created based on the passed in [urls]; local registries use the given [projectDir] as workspace.
          */
         fun create(urls: Collection<String>, projectDir: File): CompositeBazelModuleRegistryService {
-            val packageNamesForServer = urls.filter { it.endsWith("source.json") }.mapNotNull { url ->
+            val registryUrls = urls.distinctBy { it.removeSuffix("/") }
```

Comment:
I don't understand why you need disambiguation here when there is the `filter` underneath that accepts only URLs ending with `source.json`.

Reply:
Well, couldn't there still be the same URL ending with "source.json" multiple times in the collection? But anyway I agree that the `removeSuffix("/")` "normalization" can be omitted here.